tests/NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests/Features/Supplier Section Update.feature

```diff
@@ -0,0 +1,100 @@
+﻿Feature: Update an Supplier Section
+    As a Buyer
+    I want to update an supplier
+    So that I can ensure that it is kept up to date
+
+Background:
+    Given Addresses exist
+        | Line1 | Line2      | Line3      | Line4          | Line5          | Town      | County  | Postcode | Country        |
+        | 4     | Upper Flat | Rocks Lane | Little Village | Bigger Village | Some Town | W Yorks | LS 1 3AP | United Kingdom |
+    Given Contacts exist
+        | FirstName | LastName | EmailAddress            | TelephoneNumber |
+        | Fred      | Robinson | Fred.robinson@email.com | 12312543212     |
+    Given Orders exist
+        | OrderId    | OrganisationId                       | OrganisationName | OrganisationOdsCode | SupplierId | SupplierAddressPostcode | SupplierContactEmail    | Description      | OrderStatusId | Created    | LastUpdated | LastUpdatedBy                        |
+        | C000014-01 | 4af62b99-638c-4247-875e-965239cd0c48 | Hampshire CC     | 432432              | SupId      | LS 1 3AP                | Fred.robinson@email.com | Some Description | 1             | 11/05/2020 | 11/05/2020  | 335392e4-4bb1-413b-9de5-36a85c9c0422 |
+    And the user is logged in with the Buyer role for organisation 4af62b99-638c-4247-875e-965239cd0c48
+
+@4621
+Scenario: 1. Update a supplier section
+    Given the user wants to update the SupplierAddress section for the address
+        | Line1     | Line2      | Line3       | Line4          | Line5          | Town         | County  | Postcode | Country        |
+        | New Line1 | Lower Flat | Rocks Close | Larger Village | Masive Village | Another Town | N Yorks | YO11 1AP | United Kingdom |

```

Comment:
if we're being cool and saying Massive Village, let's not call it Masive Village ;)

Reply:
👍